pages/forgot-password.js

```diff
@@ -1,9 +1,103 @@
-import React from 'react'
+import React, { useState } from "react";
```

Comment:
Group import statements in this order
1. Plugin imports
2. import of our components
3. Services
4. lib / utils
5. css

Add an line break between them

Reply:
changed in commit